assets/app/views/directives/environment.html

```diff
@@ -0,0 +1,23 @@
+<div ng-if="envVars.length" class="table-responsive" style="margin-top: 5px;">
```

Comment:
Probably better to switch to a stacked representation on mobile (name above value), although that might not be simple using a table.